src/auth/s2s/s2s.class.ts

```diff
@@ -56,15 +56,15 @@ export class S2SAuth extends events.EventEmitter {
                 // If there are no listeners for a success event from this emitter, just return this middleware using
                 // next(), else emit a success event with the S2S token
                 if (!this.listenerCount(S2S.EVENT.AUTHENTICATE_SUCCESS)) {
-                    this.logger.log(`S2SAuth: no listener count: ${S2S.EVENT.AUTHENTICATE_SUCCESS}`)
+                    this.logger.info(`S2SAuth: no listener count: ${S2S.EVENT.AUTHENTICATE_SUCCESS}`)
                     return next()
                 } else {
                     this.emit(S2S.EVENT.AUTHENTICATE_SUCCESS, token, req, res, next)
                     return
                 }
             }
         } catch (error) {
-            this.logger.log('S2SAuth error:', error)
+            this.logger.error('S2SAuth error:', error.message)
```

Comment:
could we actually remove this line please? think whilst I was testing this exposed the s2s secret etc in the logs (and also again we're assuming error.mesage will actually always be present)

Reply:
I will remove this line, then